contracts/WildcardSteward_v3.sol

```diff
@@ -39,6 +38,7 @@ contract WildcardSteward_v3 is Initializable {
 
     // 1200% patronage
     mapping(uint256 => uint256) public patronageNumerator;
+    uint256 public patronageDenominator;
```

Comment:
This is is really really bad!! This would compeletly brick our contracts :(

No idea how it got forgotten!